### PR TITLE
fix Gootville vtest

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

**Comment by lasconic** [comment]
So we are storing Gonville for the musical font but Gootville Text for the text? that's weird.

**Comment by Jojo-Schmitz** [comment]
No,Gootville.oft and GootvilleTxte.otf are just the filenames, used internally, in the UI they are called Gonville and Gonville Text.
Possible that this renaming a couple weeks ago had resulted in the setting of those test files to vanish.

**Comment by lasconic** [comment]
The testfiles are currently containing reference to Gonville and Gonville Text.
The new testfiles would contain reference to Gonville and Gootville Text. Something fishy.

**Comment by Gootector** [comment]
Your problem is with font names. You are using two different names: "Gonville" and "Gootville" and I don't know - why. My fonts are named "Gootville":
Gootville.otf
GootvilleText.otf
You must fix this this problem yourself.

**Comment by lasconic** [comment]
thank you for your input...

**Comment by Gootector** [comment]
:D Hold on!

**Comment by Jojo-Schmitz** [comment]
this and the previous PR #1771 didn't change anything with regards to this. So this problem got to have been lurking since PR #1681, #1577 or #1556, the last did the renaming, the other 2 just changed the font files. None of them ever updated the vtest files.

**Comment by lasconic** [comment]
Yes, this is the problem. I think we don't get what we store in the file from the same place here.
https://github.com/musescore/MuseScore/blob/master/mscore/editstyle.cpp#L473

For score font, we store the first attribute from ScoreFont() here, which is also used in the UI. https://github.com/musescore/MuseScore/blob/master/libmscore/sym.cpp#L29 so Gonville for the Gootville font. The name of the font file is irrelevant, since it's the 3rd param in ScoreFont.

For musical text font, we store the second field from https://github.com/musescore/MuseScore/blob/master/mscore/editstyle.cpp#L740 so "Gootville Text" while the first one is used in the UI. *The second field is also used as font family name*. So here again the font filename is irrelevant but the font family in the file will make it work or not.

To sum it up, we are storing "Gootville Text" since https://github.com/musescore/MuseScore/pull/1556/files#diff-855615c9d487c6597383aed2fe12420cR740. I will annotate the code so we know all this for next time...
